src/web_app_skeleton/docker/development.dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM crystallang/crystal:1.4.1
 
+# Install utilities required to make this Dockerfile run
+RUN apt-get update && \
+    apt-get install -y wget gunzip
```

Comment:
Can this be moved down to the other section using `apt-get`?

Reply:
The line right after this runs `wget`, so it has to be installed before we get to that. Then that one is required before the following apt-get so node can be installed :sweat_smile: